### DISCUSSION TITLE
Consider adding a new badge to the README.md

### DISCUSSION THREAD

**Discussion by zaataylor** [webpack/webpack#12210] [Ideas]
Hi! 😄 

I made some badges that all kinds of open source projects can use to describe their project type, based off of the classifications from _Working in Public_ by Nadia Eghbal. I'd love it if different open source projects such as this one would actually consider adding the badges to their `README`s. You can find descriptions of the badges [here](https://project-types.github.io/), and can get a TL;DR explanation of how the use the badges [here](https://github.com/project-types/project-types.github.io).

Here are the different badges:
1. <a href="https://project-types.github.io/#toy"><img src="https://img.shields.io/badge/project%20type-toy-blue" alt="Toy Badge"></a>
2. <a href="https://project-types.github.io/#club"><img src="https://img.shields.io/badge/project%20type-club-ff69b4" alt="Club Badge"></a>
3. <a href="https://project-types.github.io/#stadium"><img src="https://img.shields.io/badge/project%20type-stadium-orange" alt="Stadium Badge"></a>
4. <a href="https://project-types.github.io/#federation"><img src="https://img.shields.io/badge/project%20type-federation-brightgreen" alt="Federation Badge"></a>

**Comment by alexander-akait** [comment]
What useful information does this band provide?

**Reply by zaataylor** [reply]
The badges/bands can be used to give potential contributors and/or users of a project an at-a-glance, rough idea of the social and organizational structure of it. For instance, someone looking at a project like [`ssh-chat`](https://github.com/shazow/ssh-chat) and seeing a <a href="https://project-types.github.io/#toy"><img src="https://img.shields.io/badge/project%20type-toy-blue" alt="Toy Badge"></a> badge on it would know that it's a "smaller" piece of software with no intention of being a huge project like webpack or babel. I think of the utility as being similar to how knowing the genre of a book might help one set their expectations for what they'll read, even if it's more of an implicit process. Classifications provide context.

Outside of this, I really only intended the badges to be an aesthetic flair that projects could add to their `README`s. My goal isn't to thrust this classification criteria on any project, so feel free to disregard it if you don't think it'll be useful. I was just really fascinated by this particular concept from _Working in Public_, and saw this as a means of realizing that concept in a more tangible way.

**Comment by rafaelfachinelli** [comment]
Perhaps this bagde will also help beginners like me to choose smaller projects to work on and train our skills.

Another point is that some people may also be at a time when they want to contribute to smaller projects because they do not have so much time available and seeing the greatness of the project quickly can help.
